utils.py

```diff
@@ -0,0 +1,64 @@
+import os
+import requests, json, time
+from collections import defaultdict
+
+from models import Package
+
+def fetch_package(org):
+    api_key = os.getenv("LIBRAIRIES_API_KEY") 
```

Comment:
I suggest replacing LIBRAIRIES_API_KEY by LIBRARIES.IO_API_KEY (at least it should be LIBRARIES, not LIBRAIRIES.)